backend/packages/Upgrade/src/api/controllers/validators/FeatureFlagsPaginatedParamsValidator.ts

```diff
@@ -12,6 +12,22 @@ export interface IFeatureFlagSortParams {
   sortAs: SORT_AS_DIRECTION;
 }
 
+export interface FeatureFlagFile {
```

Comment:
if these types are used on backend and frontend (I'd think they would be), please put them in the `types` dir

Reply:
created a common interface in upgrade_types